src/main/java/ca/uhn/fhir/jpa/starter/BackendAuthorizationInterceptor.java

```diff
@@ -49,16 +52,18 @@
                     return authorizedRule();
                 else {
                     try {
-                        TokenVerifier<AccessToken> verifier = TokenVerifier.create(token, AccessToken.class).publicKey(getKey());
-                        verifier.verify();
+                        verify(token);
                         return authorizedRule();
-                    } catch (VerificationException e) {
+                    } catch (TokenExpiredException e) {
+                        e.printStackTrace();
+                        throw new AuthenticationException("Token is expired", e);
+                    } catch (JWTVerificationException e) {
                         e.printStackTrace();
                         throw new AuthenticationException("Token is invalid", e);
                     } catch (NoSuchAlgorithmException | InvalidKeySpecException e) {
                         e.printStackTrace();
                         throw new AuthenticationException("Internal error processing public key", e);
-                    }
+					}
```

Comment:
I'm not exactly sure why this renders like that... not seeing this on my branch locally.
```suggestion
                    }
```